addons/mail/static/src/models/message/message.js

```diff
@@ -201,6 +201,22 @@ function factory(dependencies) {
             });
         }
 
+        /**
+         * Removes the message body. Unlinking the record would pose many
+         * problems in places that reference the message.
+         */
+        delete() {
```

Comment:
Did you intend to override the delete method from base model.js?

Reply:
I did not, but now that I think about it, wouldn't it be a good idea?

Reply:
No, it's technical method that shouldn't be modified. It should always be possible to delete a message in JS which is equivalent of losing knowledge of it.